tests/content/entity/zone/create/test.md

```diff
@@ -1,16 +1,39 @@
 # Entity Zone Create
```

Comment:
As we discussed last week,please keep this simple zone create test as is, thanks